### ISSUE TITLE
You miss an 'n'

### ISSUE THREAD

**Issue by ThomasWXW** [charlesq34/pointnet2#90]
Hi, 

Just want to remind that you miss an 'n' in your get_learning_rate function.

**### learing_rate** = tf.maximum(learning_rate, 0.00001) # CLIP THE LEARNING RATE!

https://github.com/charlesq34/pointnet2/blob/74bb67f3702e8aec55a7b8765dd728b18456030c/scannet/train.py#L78-L86

**Comment by chenzhutian** [comment]
PR has been raised
https://github.com/charlesq34/pointnet2/pull/101
